src/shared_realm.hpp

```diff
@@ -26,6 +26,8 @@
 #include <memory>
 #include <thread>
 
+#include <realm/group_shared.hpp>
```

Comment:
This is unfortunate as we've been trying to avoid pulling core headers into object store headers where possible.